src/components/Tabs/Tab.tsx

```diff
@@ -23,19 +23,19 @@ const Tab = styled.div`
     'selected',
     css`
       background: ${colors.white};
-      color: ${uiColors('primary.main')};
+      color: ${props => props.theme.palette.primary.main};
```

Comment:
It's good practice to destructure the theme.
```suggestion
      color: ${({ theme }) => theme.palette.primary.main};
```
The same everywhere.